docs/transactional.md

```diff
@@ -0,0 +1,31 @@
+
+# Transactional setting
+
+## Background
+As of v2.5.0 (TODO: Fix version number before release), wordpress-sparkpost introduced [Transactional](https://www.sparkpost.com/blog/transactional-email-vs-mass-email) support for emails.
+
+Emails created by WordPress are, generally, transactional in nature. Those are user registration, password reset emails etc. So, we wanted to set that flag on by default. Non-transactional emails are generated by third party plugins. In such case, that plugin should mark it as non-transactional. Because this option supports hook, it's easy to programmatically flip the value. We'll have an example below.
```

Comment:
"So, we wanted to set that flag on by default." --> "Because of this, the transactional flag defaults to true.